src/layers/MapboxStyleLayer/MapboxStyleLayer.js

```diff
@@ -151,7 +111,13 @@ class MapboxStyleLayer extends Layer {
         mbMap.addLayer(styleLayer);
       }
     });
-    applyLayoutVisibility(mbMap, this.getVisible(), this.styleLayersFilter);
+    this.applyLayoutVisibility(
+      mbMap,
+      this.getVisible(),
+      this.styleLayersFilter,
+      this.filters, // for LevelLayer
+      this.properties.radioGroup, // for LevelLayer
+    );
```

Comment:
if applyLayoutVisibility is part of the class you don't have to put parameters, just use them directly in the function.

Reply:
you never use radioGroup

Reply:
As written in the comment. it is used in LevelLayer, which overwrite the applyLayoutVisibility Method.

Reply:
Removed